### PR TITLE
Updated readme, added changes from seans PR

### PR DESCRIPTION
**Pull request by BaileyBrightman**

### CODE REVIEW

**Review by 902seanryan — APPROVED** [review]

**Comment by 902seanryan** [comment]
@BaileyBrightman sort of a weird request but I think it makes sense to get in here. Could you update the CODEOWNERS file to include @deycorinne

**Review by deycorinne — APPROVED** [review]

**Review by 902seanryan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ to `window`
 
 ## Basic Usage ##
 
-Here's a very simple example to get started. We have two pages `index.html` and `child.html`.This is the minimum you need to start get them talking to each other.
+Here's a very simple example to get started. We have two pages `index.html` and `child.html`.This is the minimum you need to get them talking to each other.
```
Comment (aberkie) [inline README.md:38]:
```suggestion
Here's a very simple example to get started. We have two pages `index.html` and `child.html`. This is the minimum you need to get them talking to each other.
```

File: README.md [hunk 2]
```diff
@@ -107,6 +107,14 @@ allows a containing page to connect with an interior iframe and emit message _in
 ### `destroy`
 `disconnect` removes any listener for events from another frame, and stops listening for messages altogether
 
+
+### `off`
+Removes an event listener previously added by the .on() method, or remove a given callback method from a listener. When deleting a callback, the function passed in is required to be the original function passed into the .on() method
```
Comment (aberkie) [inline README.md:112]:
```suggestion
Removes an event listener previously added by the `.on()` method, or removes a given callback method from a listener. When deleting a callback, the function passed in is required to be the original function passed into the `.on()` method.
```

**Review by aberkie — CHANGES_REQUESTED** [review]

**Comment by aberkie** [comment]
Added some suggestions in comments. Also, I created [this PR](https://github.com/BaileyBrightman/Bellhop/pull/1) to @BaileyBrightman's fork that adds a vscode markdown linter.

File: README.md (hunk 1 above)
Comment (902seanryan) [reply README.md:40]: @BaileyBrightman 
Just a heads up I think we lost this change in the mix somewhere.

**Review by 902seanryan — CHANGES_REQUESTED** [review]

**Review by 902seanryan — APPROVED** [review]

**Review by 902seanryan — APPROVED** [review]

**Review by deycorinne — APPROVED** [review]

**Pull request merged**